plugins.json

```diff
@@ -93,17 +93,23 @@
     "name": "Get Environment Variables",
     "package": "netlify-plugin-get-env-vars",
     "repo": "https://github.com/shortdiv/netlify-plugin-get-env-vars",
-    "version": "1.0.0",
-    "status": "DEACTIVATED"
+    "version": "1.0.0"
+  },
+  {
+    "author": "shortdiv",
+    "description": "Predictively prefetch assets. Given given a user's current route, it guesses where a user will go next. Uses GA",
+    "name": "Predictive Prefetch",
+    "package": "netlify-plugin-predictive-prefetch",
+    "repo": "https://github.com/shortdiv/netlify-plugin-predictive-prefetch",
+    "version": "1.0.1"
   },
   {
     "author": "shortdiv",
     "description": "Prerenders a SPA into separate pages. Useful for letting Netlify identify forms in a SPA",
     "name": "Prerender SPA",
     "package": "netlify-plugin-prerender-spa",
     "repo": "https://github.com/shortdiv/netlify-plugin-prerender-spa",
-    "version": "1.0.1",
```

Comment:
There are few issues opened in this plugin as well [here](https://github.com/shortdiv/netlify-plugin-prerender-spa/issues?q=is%3Aissue+is%3Aopen+author%3Aehmicky+sort%3Aupdated-desc). What do you think?